core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/dot-edit-content.component.ts

```diff
@@ -682,4 +721,54 @@ export class DotEditContentComponent implements OnInit, OnDestroy {
             })
         );
     }
+    /**
+     * Open the Dot EditContentletModeSelector component and subscribe to the onClose event
+     *
+     * @param boolean openDialog
+     * @memberof openContentletEditModeSelector
+     */
+    private openContentletEditModeSelector({
+        copyContent,
+        inode
+    }: {
+        copyContent: DotCopyContent;
+        inode: string;
+    }): void {
+        const ref = this.dialogService.open(DotBinaryOptionSelectorComponent, {
+            header: this.dotMessageService.get('Edit-Content'),
+            width: '37rem',
+            data: { options: this.CONTENT_EDIT_OPTIONS },
+            style: { 'border-radius': '16px' },
```

Comment:
Remove this, will be doing this globally for all the dialogs.

Reply:
Great, I'll do it!